tests/quick_tests/run.cmake

```diff
@@ -22,7 +22,7 @@ SEPARATE_ARGUMENTS(ALL_TESTS)
 EXECUTE_PROCESS(COMMAND ${CMAKE_CTEST_COMMAND} --force-new-ctest-process --output-on-failure -O quicktests.log RESULT_VARIABLE res_var)
 
 if(NOT "${res_var}" STREQUAL "0")
-  MESSAGE( "
+  MESSAGE(SEND_ERROR "
 
 *******************************     WARNING     *******************************
```

Comment:
if I am not mistaken @bangerth meant to rename `***** WARNING ****` to `**** ERROR ****`. As far as i understand, we should not get here unless there is an error, right?